yt_dlp/extractor/trueid.py

```diff
@@ -0,0 +1,150 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..compat import compat_HTTPError
+from ..utils import (
+    determine_ext,
+    ExtractorError,
+    int_or_none,
+    parse_age_limit,
+    traverse_obj,
+    unified_timestamp,
+    url_or_none
+)
+
+
+class TrueIDIE(InfoExtractor):
+    _VALID_URL = r'https?://(?P<domain>vn\.trueid\.net|trueid\.(?:id|ph))/(?:movie|series/[^/]+)/(?P<id>[^/?#&]+)'
+    _TESTS = [{
+        'url': 'https://trueid.id/movie/XYNlDOZZJzL6/pengabdi-setan/',
+        'md5': '2552c7535125885901f1a2a4bcf32ca3',
+        'info_dict': {
+            'id': 'XYNlDOZZJzL6',
+            'ext': 'mp4',
+            'title': 'Pengabdi Setan',
+            'display_id': 'pengabdi-setan',
+            'description': 'md5:b0b41df08601e85e5291496c9bbe52cd',
+            'timestamp': 1600243511,
+            'categories': ['Film Indonesia', 'Horror', 'Mystery'],
+            'release_timestamp': 1593536400,
+            'release_year': 1982,
+            'cast': list,
+            'thumbnail': 'https://cms.dmpcdn.com/movie/2020/09/18/8b6e35c0-f97f-11ea-81fe-c52fc9dd314f_original.png',
+            'upload_date': '20200916',
+            'release_date': '20200630',
+        },
+        'params': {
+            'format': 'bv'
+        }
```

Comment:
```py
        'params': {
            'format': 'bv'
        }
```

is not needed in tests anymore. I have changed tests' default params to handle this automatically